tools/export/gccarm.py

```diff
@@ -121,6 +121,8 @@ class GccArm(Exporter):
 
     DOT_IN_RELATIVE_PATH = True
 
+    MBED_CONF_ACTIVE = True
```

Comment:
Is this flag to signal if mbed config is active (=present) or if there is a configuration somewhere (has config)? I would say the former, same as ``PROGEN_ACTIVE`` flag. @theotherjimmy what do you think?